### ISSUE TITLE
Problem with finetuning model 

### ISSUE THREAD

**Issue by GooDRomka** [tloen/alpaca-lora#459]
I tried to fine-tune the model, took one example, and trained it for many epochs. I noticed that the loss decreased significantly, indicating that the model was learning. However, when I utilized the saved LoRa, I found that model was identical to the base model. I loaded both an overfitted model (trained for 25 epochs with 100 identical examples each) and an underfitted model (trained for 1 epoch). Surprisingly, both models produced the same text and achieved identical scores.

What are the chances that the LoRa was not saved correctly? Or maybe the problem is in the training parameters or in the training process?

**Comment by DingShi0** [comment]
I had a similar problem, and I solved it by looking at this answer. #363
You can start by looking at the size of the model file you have trained.

**Comment by GooDRomka** [comment]
Solved. Thanks
